env.ts

```diff
@@ -93,6 +94,10 @@ const envSchema = {
       __dirname,
       "../ocw-hugo-projects/ocw-www/config.yaml"
     )
+  }),
+  WWW_CONTENT_PATH: envalid.str({
+    desc:       "Path to the ocw-www Hugo configuration file",
+    devDefault: path.resolve(__dirname, "../mitocwcontent/ocw-www/config.yaml")
```

Comment:
Maybe this should default to `../ocw-content-rc/ocw-www/config.yaml` to follow suit?